geomesa-hbase/geomesa-hbase-datastore/src/test/scala/org/locationtech/geomesa/hbase/data/MiniCluster.scala

```diff
@@ -20,7 +20,7 @@ case object MiniCluster extends LazyLogging {
     logger.info("Starting embedded hbase")
     cluster.getConfiguration.set("hbase.superuser", "admin")
     cluster.getConfiguration.set(CoprocessorHost.USER_REGION_COPROCESSOR_CONF_KEY, HBaseIndexAdapter.CoprocessorClass)
-    cluster.startMiniCluster(sys.props.get("geomesa.hbase.test.servers").map(_.toInt).getOrElse(3))
+    cluster.startMiniCluster(sys.props.get("geomesa.hbase.test.region.servers").map(_.toInt).getOrElse(2))
```

Comment:
Is this system property documented / used anywhere?  Also, any particular reason for the change from 3 to 2?

Reply:
it's in the main pom.xml properties section. i actually dialed it down to 1 there, to reduce the load/complexity of the mini cluster. it seemed to improve the stability of the accumulo mini cluster immensely. 2 is the hbase default i think...